libs/api/domains/license-service/src/lib/client/machine-license-client/genericMachineLicense.config.ts

```diff
@@ -0,0 +1,18 @@
+import { defineConfig } from '@island.is/nest/config'
+import { SmartSolutionsApiConfigSchema as schema } from '@island.is/clients/smartsolutions'
+import * as z from 'zod'
+
+export const GenericMachineLicenseConfig = defineConfig<z.infer<typeof schema>>(
+  {
+    name: 'GenericMachineLicenseConfig ',
+    schema,
+    load: (env) => ({
+      apiKey: env.required('VE_PKPASS_API_KEY', ''),
```

Comment:
Having "''" as a devFallback hurts DX since the developer doesn't get notified about which environment variables they are missing. An alternative is to have no fallback but use `scope: LazyDuringDevScope`. This way the whole server doesn't crash when the environment is missing but the individual requests will fail with descriptive errors. Only during development.

Just add it to providers which depend on these partial configs (in this case SMART_SOLUTIONS_API_CONFIG).

Reply:
Yes, good point, didn't know.

Reply:
Spoke to devops, there is some weirdness in the graphql schemas generation which requires a fallback on the env.required parameters. So the fallback will stay temporarily, until they fix it at least.